python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -495,20 +497,25 @@ def _getStateTime(job, status):
             if not job.subjobs and status in b_list:
                 for childstatus in b_list:
                     if job.backend.id:
-                        logger.debug("Accessing getStateTime() in diracAPI")
-                        dirac_cmd = "getStateTime(%d,\'%s\')" % (job.backend.id, childstatus)
-                        be_statetime = execute(dirac_cmd)
+               #         logger.debug("Accessing getStateTime() in diracAPI")
+                        #be_statetime = execute("getStateTime(%d,\'%s\')" % (job.backend.id, childstatus))
```

Comment:
Ok, since you said "I'm keen to get this in at the start of 6.1.20 once the release is out" I thought this pull request was ready to be merged.